src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/CovariantEquals/newline/defaultContext.patch

```diff
@@ -1,25 +1,14 @@
 diff --git a/Test.java b/Test.java
-index cfa1cc7..6011bfc 100644
+index 6011bfc..9955098 100644
 --- a/Test.java
 +++ b/Test.java
-@@ -5,9 +5,6 @@ public class Test {
-         return false;
-     }
- 
--    public boolean equals(Object i) {
--        return false;
--    }
- }
- 
- class Test2 {
-@@ -15,7 +12,7 @@ class Test2 {
-         return false;
-     }
- 
--    public boolean equals(Object i) {
--        return false;
--    }
-+//    public boolean equals(Object i) {
-+//        return false;
-+//    }
```

Comment:
why are you deleting these changes? They are still needed to show deletion and edit.

Reply:
@baratali  these changes just on newline/defaultContext.patch, not context/defaultContext.patch, in context/defaultContext.patch, still have these.

Reply:
newline strategy should ignore deletes and edits. Why do you need to remove this code? Please restore it

Reply:
@baratali yes, deletes and edits are in newline/Test.java file, this change is newline/defaultContext.patch

Reply:
what do you mean? `newline/Test.java` is input file, it doesn't have any deletes or edits. `newline/defaultContext.patch` is a patch file and there are only new lines, because you remove other diff (https://github.com/checkstyle/patch-filters/blob/520d7b381efa92d9f05c51829a9a849268b7a93b/src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/CovariantEquals/newline/defaultContext.patch)
why do you need to remove edits/deletes from `newline/defaultContext.patch`?

Reply:
done, I remove it, because there are reasons: 1. in `newline/defaultContext.patch`, keeping deletes or edits  is to test https://github.com/checkstyle/patch-filters/blob/4f21f625945a2a36bd2acc5cd5fdc17c4d476b46/src/main/java/com/puppycrawl/tools/checkstyle/filters/LoadPatchFileUtils.java#L96-L102

Reply:
OK, I thought you were trying to workaround some bug in newline strategy, because that diff was already generated and you didn't need remove anything from it :)
Nevermind, the test looks good, I'm mering it.